lib/output.js

```diff
@@ -597,6 +598,12 @@ function OutputStream(options) {
             return true;
         }
 
+        if (output.option('webkit')
+            && output.parent(0) instanceof AST_PropAccess
+            && output.parent(1) instanceof AST_Assign) {
```

Comment:
Why is AST_Assign part of the condition at all?

The following is not handled:
```
$ echo 'console.log(function(){}.a);' | bin/uglifyjs -c -b webkit,beautify=false
console.log(function(){}.a);
```
Should be:
```js
         if (output.option("webkit")) {
             var p = output.parent();
             if (p instanceof AST_PropAccess && p.expression === this) return true;
         }
```
An aside: I noticed a few single quotes in recent commits for string literals. Generally the uglify code base uses double quotes.